lib/ostruct.rb

```diff
@@ -284,8 +284,10 @@ def dig(name, *names)
   end
 
   #
-  # Removes the named field from the object. Returns the value that the field
-  # contained if it was defined.
+  # Remove the named field from the object. Returns the value that the field
```

Comment:
Why "Remove" instead of "Removes"?
"Returns" is OK?

Reply:
I don't recall why I made that modification. I can certainly change it back. Either should work really.